.travis.yml

```diff
@@ -171,16 +171,20 @@ before_script:
       if [[ "$SKIP_BUILD" == true ]]; then
           export RUN_SCRIPT="echo 'skipping, not a full build'";
       else
-          RUN_SCRIPT="stamp src/ci/init_repo.sh . $HOME/rustsrc";
+          RUN_SCRIPT="src/ci/init_repo.sh . $HOME/rustsrc";
           if [ "$TRAVIS_OS_NAME" = "osx" ]; then
-              export RUN_SCRIPT="$RUN_SCRIPT && stamp src/ci/run.sh";
+              export RUN_SCRIPT="$RUN_SCRIPT && src/ci/run.sh";
           else
-              export RUN_SCRIPT="$RUN_SCRIPT && stamp src/ci/docker/run.sh $IMAGE";
+              export RUN_SCRIPT="$RUN_SCRIPT && src/ci/docker/run.sh $IMAGE";
           fi
       fi
 
 script:
-  - sh -x -c "$RUN_SCRIPT"
+  - >
+      date && curl -s --head https://google.com | grep ^Date: | sed 's/Date: //g'
+  - stamp sh -x -c "$RUN_SCRIPT"
+  - >
+      date && curl -s --head https://google.com | grep ^Date: | sed 's/Date: //g'
```

Comment:
It'd be good to have a comment here explaining this. I'd also like to note that comparing these two may be hard because of timezones (maybe there's something we can pass to `date`?).

I also think it may be worth printing this once before we run the script and afterwards -- that way we if we do find a problem, we know whether it happened during the build.

```
$ date && curl -s --head https://google.com | grep ^Date: | sed 's/Date: //g'
Wed Jun 28 16:24:07 MDT 2017
Wed, 28 Jun 2017 22:24:07 GMT
```